### DISCUSSION TITLE
Question about BulkMessenger

### DISCUSSION THREAD

**Discussion by Jorge-S13** [nutgram/nutgram#502] [Q&A]
I am using windows for local development. I wanted to try the Question about BulkMessenger feature, but I get the following error: RuntimeException: The pcntl extension is required. After searching a bit on google for an answer, I found out that the pcntl extension cannot be installed on windows. The question is, how can I fix this error? Or do I need to set up a Unix virtual machine to test this functionality.

**Comment by Lukasss93** [comment]
As described on the documentation [website](https://nutgram.dev/docs/usage/bulk_messenger):

![image](https://github.com/nutgram/nutgram/assets/4071613/a20cdeac-b121-45fc-a309-04f7c054df0a)

The `startAsync` method needs the pcntl extension, which is only available on Unix.
On Windows you can use a VM or Docker.

**Reply by Jorge-S13** [reply]
Thank you for your help
